### PR TITLE
Added the action build(Surface) in the environment

### PR DESCRIPTION
**Pull request by HarmGriffioen**
Added the action build(Surface) in the environment, this action can be called from GOAL

### CODE REVIEW

**Comment by devbridie** [comment]
You shouldn't have committed these files:
- `tygron-environment/demoBuildingText`
- `tygron-environment/src/GOAL/GoalTestTygron.mas2g`

File: tygron-environment/src/GOAL/GoalTestTygron.mas2g [hunk 1]
```diff
@@ -4,8 +4,8 @@ environment{
 		%stakeholder is mandatory, as is the map, if slot is not set it will result in a random slot.
 		% Stakeholder, 0 is muncipianity, 1 is the housing corporation, 2 is company
 		stakeholder = 2,
-		map = "testmap",
```
Comment (PauluzzNL) [inline tygron-environment/src/GOAL/GoalTestTygron.mas2g:7]: Pl0x this is not good :sob: Please change to "example" and the slot default to "-1" and lets leave it at that.

File: tygron-environment/demoBuildingText [hunk 2]
```diff
@@ -0,0 +1 @@
+Session link: https://server2.tygron.com:3022/api/slots/13/?token=1e594953-aae4-445e-a5c5-be71e848a29f
```
Comment (PauluzzNL) [inline tygron-environment/demoBuildingText]: I don't think we need this in our repository

**Comment by PauluzzNL** [comment]
Hatehatehatehatehate okay :+1:

**Comment by devbridie** [comment]
:+1:

**Pull request merged**